### PR TITLE
Removal of PROPKA input

### LINKED ISSUE #99: Passing a propka input to propka3 no longer works
## Issue details

As part of #98 I am testing the i/o for propka and I noticed that passing `propka_input` files generated by propka3 can no longer be read. Passing such an input will fail with the following trace:

```
Traceback (most recent call last):
  File "/home/bioc1523/software/anaconda/install/envs/propka/bin/propka3", line 33, in <module>
    sys.exit(load_entry_point('propka', 'console_scripts', 'propka3')())
  File "/home/bioc1523/github/propka/propka/run.py", line 32, in main
    my_molecule = read_molecule_file(pdbfile, my_molecule)
  File "/home/bioc1523/github/propka/propka/input.py", line 114, in read_molecule_file
    conformations, conformation_names = read_propka(
  File "/home/bioc1523/github/propka/propka/input.py", line 236, in read_propka
    for (name, atom) in lines:
  File "/home/bioc1523/github/propka/propka/input.py", line 269, in get_atom_lines_from_input
    initialize_atom_group(atom)
  File "/home/bioc1523/github/propka/propka/group.py", line 1436, in initialize_atom_group
    group_attr = globals()[atom.group_label]
KeyError: 'Nterm_group'
```

Reading these files works in 3.1, so this bug got introduced at some point in 3.2-3.3.

## Code to reproduce this behaviour

```
propka3 3SGB-subset.pdb --generate-propka-input
propka3 3SGB-subset.propka_input # fails
```

## Suspected cause

It seems like this bug might have been in part introduced in #40. However, simply changing:

https://github.com/jensengroup/propka/blob/9dfd87a0ebfd4e9e1a12149e9d8e7a621a446fc9/propka/atom.py#L325

to

            self.group_label = "{0:s}Group".format(self.occ)

won't work due to the large re-organisation of `atom.py` in #49 (one outcome of which is that `group_label` now defaults to `None`).

### PR DESCRIPTION
**Pull request by IAlibay**
Fixes #99 

### Changes made in this PR:
- Removed reading & writing of PROPKA inputs

### To do:
- Check line coverage for missed sections
- Test that docs build properly

### CODE REVIEW

File: propka/input.py [hunk 1]
```diff
@@ -67,14 +72,17 @@ def read_molecule_file(filename: str, mol_container, stream=None):
...
         >>> read_molecule_file('test.pdb', mol_container,
                                stream=string_io_object)
         <propka.molecular_container.MolecularContainer at 0x7f6e0c8f2310>
 
+
+    .. versionchanged:: 3.4.0
+       PROPKA input files (extension: `.propka_input`) are no longer read.
     """
     input_path = Path(filename)
     mol_container.name = input_path.stem
```
Comment (IAlibay) [inline propka/input.py:88]: Testing inputs by extension is probably not really necessary anymore. Thoughts about just removing this logic and just assuming everything being passed in a PDB file?

(that being said, removing the need for `filename` is going to break the API again...)

Comment (sobolevnrm) [reply propka/input.py:87]: I think we should just leave it as-is for now.

**Review by sobolevnrm — APPROVED** [review]

File: propka/atom.py [hunk 2]
```diff
@@ -33,7 +29,13 @@
...
+    """Atom class - contains all atom information found in the PDB file
+
+
+    .. versionchanged:: 3.4.0
+       :meth:`make_input_line` and :meth:`get_input_parameters` have been
+       removed as reading/writing PROPKA input is no longer supported.
+    """
 
     def __init__(self, line=None):
         """Initialize Atom object.
```
Comment (speleo3) [inline propka/atom.py:41]: `group_label`, `group_model_pka` and `group_model_pka_set` are now unused and don't need to be initialized.

Comment (IAlibay) [reply propka/atom.py:41]: Thanks, had forgotten about those, fixed in 3ca6747 :)

**Pull request merged**